### PR TITLE
Add PXF CLI test to multinode test

### PR DESCRIPTION
**Pull request by frankgh**
After the RPM work was merged, the CLI test pipeline stopped working.
This PR restores the CLI tests, but it adds the test as a task in the
multinode impersonation job. This way, it reuses the setup, which is
expensive, and then the automation tests run after the CLI tests are
successful.

### CODE REVIEW

File: concourse/scripts/cli/common.sh [hunk 1]
```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env bash
 
 export PXF_CONF=~gpadmin/pxf
-GPHOME=/usr/local/greenplum-db-devel
-export PATH=$PATH:${GPHOME}/pxf/bin
+export GPHOME=/usr/local/greenplum-db
+export PXF_HOME=$(find /usr/local/ -name "pxf-gp*" -type d)
```
Comment (bradfordb-vmware) [inline concourse/scripts/cli/common.sh:5]: I don't think we need to use `find` here, wouldn't `export PXF_HOME="/usr/local/pxf-gp*" work? If you wanted to be really defensive, you could use glob like `/usr/local/pxf-gp?` or `/usr/local/pxf-gp[56]`.

**Review by bradfordb-vmware — APPROVED** [review]

File: concourse/pipelines/templates/build_pipeline-tpl.yml [hunk 2]
```diff
@@ -1189,6 +1189,7 @@ jobs:
             segments_per_host: 4
             instance_type: n1-standard-4
             ccp_reap_minutes: 120
+            standby_master: true
```
Comment (ashuka24) [inline concourse/pipelines/templates/build_pipeline-tpl.yml:1192]: why do we need the standby master?

**Review by ashuka24 — APPROVED** [review]

File: concourse/pipelines/templates/build_pipeline-tpl.yml (hunk 2 above)
Comment (frankgh) [reply concourse/pipelines/templates/build_pipeline-tpl.yml:1192]: The PXF CLI tests make sure that standby master is also being tested

File: concourse/scripts/cli/common.sh [hunk 3]
```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env bash
 
 export PXF_CONF=~gpadmin/pxf
-GPHOME=/usr/local/greenplum-db-devel
-export PATH=$PATH:${GPHOME}/pxf/bin
+export GPHOME=/usr/local/greenplum-db
+export PXF_HOME=/usr/local/pxf-gp?
```
Comment (denalex) [inline concourse/scripts/cli/common.sh:5]: is `?` supposed to be there ?

File: concourse/pipelines/templates/dev_build_pipeline-tpl.yml [hunk 4]
```diff
@@ -1074,6 +1075,12 @@ jobs:
       GP_VER: [[gp_ver]]
       KERBEROS: ((kerberos-enabled))
       PXF_JVM_OPTS: ((pxf-jvm-opts))
+  - task: Test PXF CLI
+    image: gpdb[[gp_ver]]-pxf-dev-centos7-image
+    file: pxf_src/concourse/tasks/test_pxf_cli.yml
+    params:
+      TARGET_OS: centos
+      TARGET_OS_VERSION: 7
```
Comment (denalex) [inline concourse/pipelines/templates/dev_build_pipeline-tpl.yml:1083]: does this assume CLI tests leave PXF installation / configuration in a correct state for the downstream feature suite to work ? If yes, that would be a burden for future CLI test developers not aware of this coupling. If not, should the feature suite cleanup everything before starting ?

Comment (frankgh) [reply concourse/pipelines/templates/dev_build_pipeline-tpl.yml:1083]: that's a concern I have in the back of my mind. I was considering moving this task to run after the automation test task. That way we don't need to worry about the state of PXF before running the automation tests

File: concourse/scripts/cli/common.sh (hunk 3 above)
Comment (frankgh) [reply concourse/scripts/cli/common.sh]: it turns out glob expansion doesn't work when defining env variables

File: concourse/pipelines/templates/dev_build_pipeline-tpl.yml (hunk 4 above)
Comment (frankgh) [reply concourse/pipelines/templates/dev_build_pipeline-tpl.yml]: ok, moved the test to run as the last task.

File: concourse/scripts/cli/common.sh (hunk 1 above)
Comment (frankgh) [reply concourse/scripts/cli/common.sh:5]: I couldn't get globs working here. Reverted back to find

**Review by denalex — APPROVED** [review]

**Pull request merged**